msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,314 @@
+### YamlMime:Tutorial
+title: Teams File Upload Bot
```

Comment:
Change the title to Upload files from bot

Reply:
changes made

Reply:
changes made